packages/overlay/test/overlay.test.ts

```diff
@@ -833,8 +835,9 @@ describe('Overlay - timing', () => {
                 },
             ],
         });
-        await nextFrame();
-        await nextFrame();
+        // allow "hover" content to close by waiting for its timer to complete
+        await aTimeout(400);
```

Comment:
If the tooltip should be “closed” at this point, could we leverage an `sp-closed` listener instead?